app/models/legacy-base.model.js

```diff
@@ -65,6 +65,159 @@ class LegacyBaseModel extends BaseModel {
   static get schema () {
     throw new Error('defaultSchema() not implemented in child class')
   }
+
+  /**
+   * Translations for non-standard column names to our standardised ones
+   *
+   * The primary example of non-standard column names in the legacy data is timestamps. For example, we have found the
+   * column which has the timestamp for when a record is created has been named `date_created`, 'created', 'modified',
+   * and 'created_at`.
+   *
+   * Within our code we want to just be using `created_at`, in this example, as this is what the field would have been
+   * called if Knex had been used to generate the migrations and build the tables.
+   *
+   * All models that extend `LegacyBaseModel` are expected to implement this getter. It should always return an array.
+   * If there are no translations an empty array is valid.
+   *
+   * @example
+   * ```javascript
+   * static get translations () {
+   *  return [
+   *    { database: 'created', model: 'createdAt' },
+   *    { database: 'modified', model: 'updatedAt' }
+   *  ]
+   *}
```

Comment:
This is really picky but once I saw it I couldn't unsee it! It just needs a space to line up with L84
```suggestion
   * }
```